src/hook/useNion.js

```diff
@@ -8,195 +9,178 @@ import nionActions from '../actions'
 import { INITIALIZE_DATAKEY, UPDATE_REF } from '../actions/types'
 import { makeRef } from '../transforms'
 
-function useNion(declaration, deps = []) {
+import { areMergedPropsEqual } from '../decorator/should-rerender'
+
+export const ERROR_INVALID_NION_ACTION = 'Invalid Nion action'
+
+function coerceDeclaration(declaration) {
+    return typeof declaration === 'string'
+        ? { dataKey: declaration }
+        : declaration
+}
+
+function getOptions(decl, params, actionOptions, body) {
+    const declaration = coerceDeclaration(decl)
+
+    const o = {
```

Comment:
same comment as on the c/component etc thing: just a little more clarity on the variable name. Even calling it 'options' on line 25 tells me a whole lot more of what to expect about how this variable will be used in the 'getOptions' function. I suppose 'o' hints at it, but it also reminds me of IE3 Jscript documentation where 'o' would be used for any object.

Reply:
For the one object being constructed in a function called `getOptions`, is being that verbose necessary?

Reply:
I don't know it's the one object being returned until line 43.